transact/build.gradle.kts

```diff
@@ -43,6 +43,7 @@ dependencies {
     testImplementation("io.rest-assured:rest-assured:5.4.0")
     testImplementation("io.rest-assured:json-path:5.4.0")
     testImplementation("io.rest-assured:xml-path:5.4.0")
+    testImplementation("org.testcontainers:postgresql:1.21.3")
```

Comment:
As per https://java.testcontainers.org/modules/databases/postgres/, this appears to be an outdated version